Generator/Sources/NeedleFramework/Models/Pluginized/PluginExtension.swift

```diff
@@ -16,6 +16,14 @@
 
 import Foundation
 
+/// The data model representing a plugin extension provider to be generated for a
+/// specific pluginized component.
+struct PluginExtensionProvider {
+    let coreName : String
+    let nonCoreName : String
```

Comment:
These should probably be named `pluginizedComponentName` and `nonCoreComponentName`.

Reply:
unused struct, deleted.